integration-tests/pom.xml

```diff
@@ -18,10 +18,10 @@
       <groupId>io.quarkiverse.operatorsdk</groupId>
       <artifactId>quarkus-operator-sdk</artifactId>
     </dependency>
-    <dependency>
+   <!-- <dependency>
       <groupId>io.quarkiverse.operatorsdk</groupId>
       <artifactId>quarkus-operator-sdk-bundle-generator</artifactId>
-    </dependency>
+    </dependency>-->
```

Comment:
intentional or forgotten test?

Reply:
Good point! Need to double check but if it works nicely without, might as well keep it removed since this part shouldn't be checking the bundle generation and therefore could be more efficient without that useless processing.